packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -250,6 +257,10 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 					minTTLInSeconds: pendingEntry.minTTLInSeconds,
 					expired,
 				});
+				if(expired) {
+					this.deleteAndEmitsIfEmpty(localId);
+					throw new Error("Trying to send BlobAttachOp of expired blob");
```

Comment:
I think it might be better to explicitly close the container here, or maybe assert since we don't expect it to happen. Closing would prevent us from submitting ops that reference this blob. I'm not really sure where this error will go.